TechnologyAssessmentApp/TechnologyAssessmentApp/POP Pattern/Shakeable.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Shakeable.swift
+//  TechnologyAssessmentApp
+//
+//  Created by Kshitij Godara on 24/08/18.
+//  Copyright © 2018 Nagarro. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol Shakeable { }
```

Comment:
Ask yourself what's the difference between `Shakeable` and `ShakeDeviceProtocol`.